pyproject.toml

```diff
@@ -41,10 +41,15 @@ pytest-cov = "*"
 requests-mock = "*"
 mypy = "^1.5.1"
 types-Jinja2 = "^2.11.9"
-types-pkg-resources = "^0.1.3"
+types-setuptools= ">=75.5.0.20241116"
```

Comment:
`types-pkg-resources` was completely removed from pypi--all versions. Was fairly annoyed by that.